repository/p.json

```diff
@@ -2719,6 +2719,18 @@
 				}
 			]
 		},
+		{
+			"name": "PSS",
+			"details": "https://github.com/blutsvente/PSS-DSL-syntax",
+			"issues": "https://github.com/blutsvente/PSS-DSL-syntax/issues",
```

Comment:
That’s not necessary, package control understands normal GitHub urls and will find the issues without you specifying it here.